index.dd

```diff
@@ -126,7 +126,7 @@ $(P Automatic memory management makes for safe, simple, and robust code.
 D also supports scoped resource management (aka the
 $(HTTP en.wikipedia.org/wiki/Resource_Acquisition_Is_Initialization, RAII) idiom)
 and $(LINK2 statement.html#ScopeGuardStatement, $(D scope) statements) for
-deterministic transactional code that is easy to write and read. $(EXAMPLE 2,
+deterministic, transactional code that is easy to write and read. $(EXAMPLE 2,
```

Comment:
I think adding the comma would be wrong. I don't think the sentence is supposed to say that D allows for transactional code, but rather that D allows such code to be deterministic.